bool data
float64 latitude
float64 longitude
---
bool status
string element
float64 mean
float64 error
string file_name
